src/main/java/in/twizmwaz/cardinal/module/modules/damageIndicator/DamageIndicator.java

```diff
@@ -0,0 +1,152 @@
+package in.twizmwaz.cardinal.module.modules.damageIndicator;
+
+import com.google.common.base.Optional;
+import in.twizmwaz.cardinal.Cardinal;
+import in.twizmwaz.cardinal.GameHandler;
+import in.twizmwaz.cardinal.event.MatchEndEvent;
+import in.twizmwaz.cardinal.event.MatchStartEvent;
+import in.twizmwaz.cardinal.event.PlayerChangeTeamEvent;
+import in.twizmwaz.cardinal.module.Module;
+import in.twizmwaz.cardinal.module.modules.team.TeamModule;
+import in.twizmwaz.cardinal.settings.Settings;
+import in.twizmwaz.cardinal.util.PacketUtils;
+import in.twizmwaz.cardinal.util.Teams;
+import net.minecraft.server.DataWatcher;
+import net.minecraft.server.DataWatcherRegistry;
+import net.minecraft.server.PacketPlayOutEntity;
+import net.minecraft.server.PacketPlayOutEntityDestroy;
+import net.minecraft.server.PacketPlayOutSpawnEntityLiving;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.event.player.PlayerQuitEvent;
+import org.bukkit.util.Cuboid;
+import org.bukkit.util.Ray;
+import org.bukkit.util.Vector;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.UUID;
+
+public class DamageIndicator implements Module {
+
+    public static double width = 0.75;
```

Comment:
these fields should be made final constants. Constant names are usually written in caps separated by underscores